### PR TITLE
Update sonobuoy to version 0.20.0

### PR DESCRIPTION
**Pull request by rhrazdil**
Fix warning:
`time="2021-03-05T10:42:11Z" level=warning msg="The maximum supported Kubernetes version is 1.19.99, but the server version is v1.20.4. Sonobuoy will continue but unexpected results may occur."`

Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

### CODE REVIEW

**Review by ormergi — APPROVED** [review]
Thanks!

**Comment by rhrazdil** [comment]
rebased

File: cluster-provision/k8s/1.20/conformance.json [hunk 1]
```diff
@@ -1 +1,95 @@
...
+    "timeoutseconds": 10800
+  },
+  "Plugins": [
+    {
+      "name": "e2e"
+    },
+    {
+      "name": "systemd-logs"
+    }
+  ],
```
Comment (ormergi) [inline cluster-provision/k8s/1.20/conformance.json:84]: Please change the plugins array to be empty, we controll which plugins to run from `check-cluster-up.sh`
https://github.com/kubevirt/kubevirtci/commit/d0fe64f8246e4a10b58f05ed9ab77e8743dc735f

**Review by ormergi — CHANGES_REQUESTED** [review]

File: cluster-provision/k8s/1.20/conformance.json (hunk 1 above)
Comment (rhrazdil) [reply cluster-provision/k8s/1.20/conformance.json]: Thank you, I missed this change. Done

File: cluster-provision/k8s/1.20/conformance.json [hunk 2]
```diff
@@ -1 +1,88 @@
...
+    "timeoutseconds": 10800
+  },
+  "Plugins": null,
+  "PluginSearchPath": [
+    "./plugins.d",
+    "/etc/sonobuoy/plugins.d",
+    "~/sonobuoy/plugins.d"
+  ],
+  "Namespace": "sonobuoy",
+  "WorkerImage": "sonobuoy/sonobuoy:v0.20.0",
```
Comment (ormergi) [inline cluster-provision/k8s/1.20/conformance.json:84]: Sorry I missed that one, please change to `projects.registry.vmware.com/sonobuoy/sonobuoy:v0.20.0`

**Review by ormergi — CHANGES_REQUESTED** [review]

File: cluster-provision/k8s/1.20/conformance.json (hunk 2 above)
Comment (rhrazdil) [reply cluster-provision/k8s/1.20/conformance.json]: Thanks, I diffed  `k8s/1.20/conformance.json` and  `k8s/1.19/conformance.json` to make sure the file are the same

**Review by ormergi — APPROVED** [review]
Thanks!

**Pull request closed without merging**